broker/message.proto

```diff
@@ -12,4 +12,5 @@ message Message {
 	string protocol    = 4;
 	string contentType = 5;
 	bytes  payload     = 6;
+	int64  created     = 7;
```

Comment:
Instead of moving Unix nanoseconds though this field, I would suggest protobuf v3 [Timestamp](https://developers.google.com/protocol-buffers/docs/reference/csharp/class/google/protobuf/well-known-types/timestamp), which is self-documenting and can be decoded in all protobuf-supported languages.